app/assets/sass/application.scss

```diff
@@ -645,3 +648,62 @@ ul.sub-tabs li a.active span.badge {
   margin: 30px 0 5px 0;
   padding: 15px 0 0 0;
 }
+
+.govuk-hint {
+    font-size: 19px;
+}
+
+.govuk-inset-text {
+    font-size: 19px;
+}
+
+.govuk-body {
+    font-size: 19px;
+}
+
+.govuk-back-link {
+    font-size: 16px;
+}
+
+.govuk-input {
```

Comment:
Overriding a lot of GOV.UK defaults here. Does this not affect other, existing pages?

May be SASS variables for some of these overrides (https://frontend.design-system.service.gov.uk/sass-api-reference/); or possibly using some of the override classes for particular elements that need it 🤷‍♂️

Reply:
I check

> Overriding a lot of GOV.UK defaults here. Does this not affect other, existing pages?
> 
> May be SASS variables for some of these overrides (https://frontend.design-system.service.gov.uk/sass-api-reference/); or possibly using some of the override classes for particular elements that need it 🤷‍♂️

I check and seems no impact as these are govuk component taken from govuk-frontend. Somehow these were getting overriden by some existing but these are not used in existing so seems all ok.